src/deploy/functions/build.ts

```diff
@@ -360,40 +348,43 @@ function discoverTrigger(endpoint: Endpoint): backend.Triggered {
     const bkEventFilters: Record<string, string> = {};
     for (const key in endpoint.eventTrigger.eventFilters) {
       if (typeof key === "string") {
```

Comment:
It can't, but the check was required because I was using `in` instead of `of` in the loop; with that fixed this check is no longer necessary and has been removed.